multiraft/multiraft.go

```diff
@@ -531,8 +625,16 @@ func (s *state) handleWriteResponse(response *writeResponse, readyGroups map[uin
 				}
 				log.V(3).Infof("node %v applying configuration change %v", s.nodeID, cc)
 				// TODO(bdarnell): dedupe by keeping a record of recently-applied commandIDs
-				// TODO(bdarnell): support removing nodes; fix double-application of initial entries
-				err = s.addNode(cc.NodeID)
+				switch cc.Type {
+				case raftpb.ConfChangeAddNode:
+					err = s.addNode(cc.NodeID, groupID)
+				case raftpb.ConfChangeRemoveNode:
+					// TODO(bdarnell): support removing nodes; fix double-application of initial entries
+					continue
+				case raftpb.ConfChangeUpdateNode:
+					// TODO(Tobias): What can be updated in such a request?
```

Comment:
I'm not sure. This may be for updating the cc.Context field (which we don't use). I don't think we need to support this.